src/main/java/org/vafer/jdeb/ControlBuilder.java

```diff
@@ -103,13 +104,19 @@ void buildControl(BinaryPackageControlFile packageControlFile, File[] controlFil
             if ("conffiles".equals(file.getName())) {
                 foundConffiles = true;
             }
-            
             if (CONFIGURATION_FILENAMES.contains(file.getName()) || MAINTAINER_SCRIPTS.contains(file.getName())) {
                 FilteredFile configurationFile = new FilteredFile(new FileInputStream(file), resolver);
                 configurationFile.setOpenToken(openReplaceToken);
                 configurationFile.setCloseToken(closeReplaceToken);
                 addControlEntry(file.getName(), configurationFile.toString(), outputStream);
 
+            } else if (file.getName().endsWith(".bin")) {
```

Comment:
I guess the proper way to fix this is expanding https://github.com/tcurdt/jdeb/blob/master/src/main/java/org/vafer/jdeb/utils/InformationInputStream.java

Right now we are only checking for the BOM and the shebang. We could also also try to detect a binary there.